docs/model-dev-guide/api-guides/batch-processing/batch-process-api-ug.rst

```diff
@@ -169,19 +210,54 @@ Initialize the dataset you want to process.
            root="/data", train=False, download=True, transform=transform
        )
 
-Step 3: Pass the InferenceProcessor Class and Dataset
+Step 4: Pass the InferenceProcessor Class and Dataset
 =====================================================
 
-Finally, pass the InferenceProcessor class and the dataset to ``torch_batch_process``.
+Pass the InferenceProcessor class and the dataset to ``torch_batch_process``.
 
 .. code:: python
 
    """
    Pass processor class and dataset to torch_batch_process
    """
    torch_batch_process(
-           InferenceProcessor,
-           dataset,
-           batch_size=64,
-           checkpoint_interval=10
+        InferenceProcessor,
+        dataset,
+        batch_size=64,
+        checkpoint_interval=10
+    )
+
+(Optional) Step 5: Send and Query Custom Inference Metrics
+==========================================================
```

Comment:
Step 5: Send and Query Custom Inference Metrics (Optional) 
==========================================================